core-web/libs/block-editor/src/lib/extensions/asset-uploader/components/upload-placeholder/upload-placeholder.component.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, EventEmitter, Output, Input } from '@angular/core';
+
+import { ButtonModule } from 'primeng/button';
+
+import { DotSpinnerModule } from '@dotcms/ui';
+
+@Component({
+    selector: 'dot-upload-placeholder',
+    templateUrl: './upload-placeholder.component.html',
+    styleUrls: ['./upload-placeholder.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    standalone: true,
+    imports: [CommonModule, ButtonModule, DotSpinnerModule]
```

Comment:
Instead `CommonModule` you can import just what you need.

Reply:
I imported `CommonModule` because I'm using a `ngSwitch` directive in the template.